### PR TITLE
add coala-analyzer.yml

### PR DESCRIPTION
**Pull request by Udayan12167**

### CODE REVIEW

File: _data/projects/coala-analyzer.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+- newcomer friendly
+- code analysis
+- linters
+- python3
+- cli
+- automation
+- devops
+upforgrabs:
+  name: difficulty/newcomer
+  link: https://github.com/coala-analyzer/coala/labels/difficulty%2Fnewcomer
```
Comment (sils) [inline _data/projects/coala-analyzer.yml]: @Udayan12167 let's use http://tinyurl.com/coala-new here instead, that works for all coala projects

File: _data/projects/coala-analyzer.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+tags:
+# Note those are tags categorizing your project, not issue labels.
+- python
+- newcomer friendly
+- code analysis
+- linters
+- python3
+- cli
+- automation
+- devops
```
Comment (sils) [inline _data/projects/coala-analyzer.yml:13]: maybe add `framework`?

File: _data/projects/coala-analyzer.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: coala-analyzer
```
Comment (sils) [inline _data/projects/coala-analyzer.yml]: can we use the `coala` name? We only have coala-analyzer on github as coala was unavailable

**Comment by sils** [comment]
@Udayan12167 can you rename the yml to just coala?

**Comment by shiftkey** [comment]
> can you rename the yml to just coala?

This will break the site generation code.

**Comment by sils** [comment]
ah it has to be the github org name?

**Comment by shiftkey** [comment]
@sils1297 no, the filename isn't really checked beyond it needing a `.yml` extension - but having something recognisable definitely helps!

File: _data/projects/coala-analyzer.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: coala
+desc: coala provides a common multiplatform interface for linting and fixing all your code, regardless of the programming language you use. Its convenient user interface eliminates a lot of common tasks for the developer. coala also facilitates people entering the world of open source by providing them valuable feedback on coding standards and formatting. So, in a nutshell coala is a useful abstraction for static code analysis that is applicable to any text based language.
```
Comment (shiftkey) [inline _data/projects/coala-analyzer.yml:2]: This is probably a bit long, but I'm going to merge it in anyway - feel free to tweak this once the site is published in a few minutes.

**Pull request merged**